test/component/uk/gov/hmrc/exports/steps/SubmissionSteps.scala

```diff
@@ -138,9 +138,9 @@ object `Submission was updated for mrn` extends Postcondition with Eventually {
 
   override def execute(context: ScenarioContext): ScenarioContext = {
     val repo = context.get[SubmissionRepository]
-    val notification = context.get[Notification]
-    val converstationId = context.get[String]
-    eventually { verify(repo).updateMrn(eqm(converstationId), eqm(notification.mrn)) }
+    val notification: Notification = context.get[Notification]
+    val conversationId = context.get[String]
+    eventually { verify(repo).updateMrn(eqm(conversationId), eqm(notification.mrn)) }
```

Comment:
`notification.mrn` is broken. I do not know enough reg. what is happening here. Any help in fixing this? Thanks.

Reply:
btw... how is this `component` folder supposed to be executed?

Reply:
I thought it should be `component:test` but that don't seem to work :thinking:

Reply:
Fixed it by adding `.settings(inConfig(ComponentTest)(Defaults.testSettings): _*)` to line #58 in build.sbt
Mrn has moved to the `NotificationDetails` class inside `Notification`.

Reply:
Sorry... I did not explain my prob clearly. I saw that `mrn` is now a field in `NotificationDetails`, but the latter is an `Option` in `Notification` and I do not know how this test might be fixed when `NotificationDetails` is `None`.